src/common/meta/src/key/flow/table_flow.rs

```diff
@@ -238,6 +238,25 @@ impl TableFlowManager {
 
         Ok(Txn::new().and_then(txns))
     }
+
+    async fn remap_table_flow_addresses(
+        &self,
+        table_flows: &mut [(TableFlowKey, TableFlowValue)],
+    ) -> Result<()> {
+        let keys = table_flows
+            .iter()
+            .map(|(_, value)| NodeAddressKey::with_flownode(value.peer.id))
+            .collect::<Vec<_>>();
+        let flownode_addrs =
+            flownode_addr_helper::get_flownode_addresses(self.kv_backend.clone(), keys).await?;
+        for (_, table_flow_value) in table_flows.iter_mut() {
+            let flownode_id = table_flow_value.peer.id;
+            if let Some(flownode_addr) = flownode_addrs.get(&flownode_id) {
+                table_flow_value.peer.addr = flownode_addr.peer.addr.clone();
+            }
```

Comment:
What if no corresponding node address for given node id? Should we consider it as a bug?

Reply:
If an id lacks a corresponding address in the `flow_node_addrs` map, it means the address hasn't been updated yet. In this case, the old address in `table_flow_value` is still valid, which is expected.